src/Main.java

```diff
@@ -1,9 +1,40 @@
+    import java.util.Scanner;
 
 public class Main {
     public static void main(String[] args) {
 
-        // Hier mag je je code schrijven voor de hoofd-opdracht
+        Integer[] numeric = {1, 2, 3, 4, 5, 6, 7, 8, 9, 0};
+        String[] alphabetic = {"one", "two", "three", "four", "Five", "six", "seven", "eight", "nine", "zero"};
 
+        Translator translate = new Translator(alphabetic, numeric);
 
+        Boolean play = true;
+        String invalid = "invalid input";
+
+        Scanner scanner = new Scanner(System.in);
+
+        while (play) {
```

Comment:
Persoonlijk zou ik ervoor kiezen dit soort lappen code in een aparte methode te zetten om het overzicht te bewaren